### DISCUSSION TITLE
Table 1 Demographics 만드는 법

### DISCUSSION THREAD

**Discussion by shanmdphd** [pipetcpt/trainees#32] [General]
```r
library(table1) # install.packages('table1')
library(moonBook) # install.packages('moonBook')

table1(~ age + sex + smoking | Dx + cardiogenicShock, data = moonBook::acs)
```

<img width="1228" alt="image" src="https://github.com/pipetcpt/trainees/assets/6946821/b722b62d-d49a-48ea-af14-974a8e616137">

논문의 Table 1은 주로 demographics를 기재합니다. 이것을 만드는 것을 도와주는 패키지와 함수 사용법 예시입니다.

**Comment by shanmdphd** [comment]
@kindasoo acs 자료의 여러 column을 통해 table 1 더 만들어보기

**Reply by KINDASOO** [reply]
![image](https://github.com/pipetcpt/trainees/assets/162951113/530a2134-d7a4-4378-814e-4ff3692752f7)

**Reply by shanmdphd** [reply]
굿!! 수고했어요. 코드도 올려주세요

**Reply by KINDASOO** [reply]
```
install.packages('table1')
install.packages('moonBook')
library(table1) 
library(moonBook) 

table1(~ age + sex + BMI + cardiogenicShock + DM + HBP + smoking | Dx, data = moonBook::acs)
```

**Comment by shanmdphd** [comment]
```r

table1(~ age + sex + BMI + cardiogenicShock + entry + 
         EF + height + weight + obesity + TC + 
         LDLC + HDLC + TG + DM + HBP + 
         smoking | Dx, data = moonBook::acs)
```

<img width="473" alt="image" src="https://github.com/pipetcpt/trainees/assets/6946821/8705bfee-a925-4430-b08c-9b69ea7a8541">
